.gitignore

```
@@ -3,4 +3,13 @@ node_modules
 **/cypress/videos
 **/cypress/report.json
 **/cypress/screenshots/actual/
-.DS_Store
\ No newline at end of file
+.DS_Store
+**/test-results/
+**/playwright-report/
+**/blob-report/
+**/playwright/.cache/
+**/test-results/
+**/playwright-report/
+**/blob-report/
+**/playwright/.cache/
+**/playwright/.auth/
```

I'm possibly missing something but I can't see anything added in this PR that lives in any of these directories plus there seems to be a bit of repetition. Maybe we can trim this down to the following?

```suggestion
**/blob-report/
**/playwright-report/
**/playwright/
**/test-results/
```